features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/APIClient.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
Why do we need to disable the rule for the entire file?

Reply:
Fixed with `a25e819`